src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/AdditionalFactorsForWomen.kt

```diff
@@ -60,7 +60,7 @@ class AdditionalFactorsForWomen(
       .filter { qualifyingConvictions(it.sentence) }
```

Comment:
For me a value of the service is that there can be a `getBreachedConvictions` method, and this method becomes the business rule only  `if` any() 2 : 0`